README.md

```diff
@@ -41,6 +41,9 @@ gem install strong_csv
 TBD: This hasn't yet been implemented.
 
 ```ruby
+require "strong_csv"
+using StrongCSV::Types::Literal # This is required to use Union types for literal values.
```

Comment:
I'm a bit skeptical about this implementation. It might be inconvenient to make developers use `using`. However, I absolutely want to avoid monkey-patch for built-in classes, so it'd be inevitable to me.